src/main.c

```diff
@@ -193,8 +193,8 @@ UX_FLOW(
 );
 
 void display_settings() {
-  strcpy(confirmLabel, (N_storage.dataAllowed ? "Allowed" : "NOT Allowed"));
-  ux_flow_init(0, ux_settings_flow, NULL);
+    strlcpy((char *) confirmLabel, (N_storage.dataAllowed ? allowed : not_allowed),32);
```

Comment:
Where are `allowed` and `not_allowed" defined? 
Also use `sizeof(confirmLabel)` instead of `32`.

Reply:
No it does not build. Fixing tat now. Needs to go back to inline static chars.